cupyx/scipy/sparse/linalg/__init__.py

```diff
@@ -7,3 +7,4 @@
 from cupyx.scipy.sparse.linalg._eigen import eigsh  # NOQA
 from cupyx.scipy.sparse.linalg._eigen import svds  # NOQA
 from cupyx.scipy.sparse.linalg._iterative import cg  # NOQA
+from cupyx.scipy.sparse.linalg.interface import *  # NOQA
```

Comment:
Conventionally we don't use `__all__`. Could you list here the interfaces exposed to users?
```suggestion
from cupyx.scipy.sparse.linalg.interface import LinearOperator  # NOQA
from cupyx.scipy.sparse.linalg.interface import aslinearoperator  # NOQA
```